### PR TITLE
ucpc-solutions:0.1.0

### PR DESCRIPTION
**Pull request by ShapeLayer**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: ucpc-solutions is the template for solutions editorial of algorithm contests, used widely in the ["Baekjoon Online Judge"](https://acmicpc.net/) users community in Korea. The original version of ucpc-solution is written in LaTeX([ucpcc/2020-solutions-theme](https://github.com/ucpcc/2020-solutions-theme)), and this is the port of LaTeX ver. This contains content-generating utils for making solutions editorial and ["solved.ac"](https://solved.ac/) difficulty expression presets, a rating system for Baekjoon Online Judge's problems.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/ucpc-solutions/0.1.0/docs/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+```typst
+#import "<template.typ>": *
+#show: <template>.with()
+= Document Title
+```
+
+ucpc-solutions requires calling a somewhat complex function.
+
+```typst
+#import "/lib/lib.typ" as ucpc
```
Comment (elegaanz) [inline packages/preview/ucpc-solutions/0.1.0/docs/README.md:20]:
```suggestion
#import "@preview/ucpc-solutions:0.1.0" as ucpc
```

File: packages/preview/ucpc-solutions/0.1.0/template/main.typ [hunk 2]
```diff
@@ -0,0 +1,94 @@
+#import "/lib/lib.typ" as ucpc
```
Comment (elegaanz) [inline packages/preview/ucpc-solutions/0.1.0/template/main.typ:1]: For this import to work when people will create their project from your template, it should use the package name.

```suggestion
#import "@preview/ucpc-solutions:0.1.0" as ucpc
```

**Comment by ShapeLayer** [comment]
Thank you for the reviews!

**Comment by elegaanz** [comment]
Thank you for taking them into account (and for the package submission) :)

**Pull request merged**